.github/workflows/note-arduino-ci.yml

```
@@ -18,3 +18,9 @@ jobs:
       - name: Run Tests In Container
         id: containerized_tests
         uses: ./.github/actions/run-tests-in-container
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - name: Publish test coverage
```

This is the name that shows on GitHub, and typical title casing would be:

`Publish Test Coverage`
```suggestion
      - name: Publish Test Coverage
```